examples/live/make_singles_fits_file.py

```diff
@@ -0,0 +1,63 @@
+"""
```

Comment:
Just making sure I understand, this file is used to calculate the significance of single detector triggers? It represents the results of fitting an exponential to the background distribution of noise triggers?

Reply:
I think @titodalcanton has the correct rights